lib/api/src/grpc/validate.rs

```diff
@@ -176,12 +176,70 @@ pub fn validate_named_vectors_not_empty(
     Err(err)
 }
 
+/// Validate the collection name contains no illegal characters.
+pub fn validate_collection_name(value: &str) -> Result<(), ValidationError> {
```

Comment:
I think there was a bad merge, I do not think you introduced this.

Reply:
This function now lives in `lib/common/validation`

Reply:
removed.